internal/kafka/internal/services/util_test.go

```diff
@@ -425,7 +426,7 @@ func Test_BuildCustomClaimCheck(t *testing.T) {
 	RegisterTestingT(t)
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			receivedBuiltCustomClaimCheck := BuildCustomClaimCheck(tt.args.kafkaRequest, tt.args.ssoconfigProvider)
+			receivedBuiltCustomClaimCheck := BuildCustomClaimCheck(tt.args.kafkaRequest, string(tt.args.ssoconfigProvider))
```

Comment:
Is the type cast needed if the type is also being changed to string on line 401?

Reply:
@dhirajsb Could you give this another review please when you have the time.

Reply:
Yes in my opinion as the scope of this PR has been reduced significantly following other earlier changes around this.